conda/base/context.py

```diff
@@ -201,6 +201,7 @@ class Context(Configuration):
     remote_connect_timeout_secs = PrimitiveParameter(9.15)
     remote_read_timeout_secs = PrimitiveParameter(60.)
     remote_max_retries = PrimitiveParameter(3)
+    remote_backoff_factor = PrimitiveParameter(0)
```

Comment:
Could this default value be changed to 1 so that users can benefit from this change by default.

Reply:
This is a great suggestion. 😀
I will change the default value to 1.
( I set it to 0 originally because it is the default value in ``Retry`` class. 😀)
Thank you for giving me suggestion. 😊